rest_framework-stubs/serializers.pyi

```diff
@@ -171,9 +165,6 @@ class Serializer(
 
 class ListSerializer(
     BaseSerializer[
-        List[Any],
-        Union[MultiValueDict, List[Any]],
-        List[Dict[str, Any]],
         List[Dict[str, Any]],
```

Comment:
same problem here, no need to make such constraints on user, for example I can pass QuerySet as instance, while you strictly require List of dicts for some reason.

Reply:
also why in init `instance: Union[MultiValueDict, Sequence[Any]] = ...,`?

Reply:
This is based on the DRF tests and in codes comments.

Reply:
changed